src/reflect/scala/reflect/internal/Mode.scala

```diff
@@ -72,10 +72,12 @@ object Mode {
    */
   final val TYPEPATmode: Mode   = 0x10000
 
+  final val SELQUALmode: Mode   = 0x20000
+
   private val StickyModes: Mode       = EXPRmode | PATTERNmode | TYPEmode
   private val StickyModesForFun: Mode = StickyModes | SCCmode
-  final val MonoQualifierModes: Mode  = EXPRmode | QUALmode
-  final val PolyQualifierModes: Mode  = EXPRmode | QUALmode | POLYmode
+  final val MonoQualifierModes: Mode  = EXPRmode | QUALmode | SELQUALmode
+  final val PolyQualifierModes: Mode  = MonoQualifierModes | POLYmode
   final val OperatorModes: Mode       = EXPRmode |            POLYmode | TAPPmode | FUNmode
```

Comment:
`OperatorModes` grid format doesn't align `POLY` any more. We need a scalafix. Maybe instead of whitespace it needs a `NONEmode` placeholder.